docker-compose/README.md

```diff
@@ -0,0 +1,11 @@
+# ewasm Testnet Local Configuration
+## Setup
```

Comment:
Can you add a note for system requirements? e.g. Linux only, etc. (`build.sh` definitely won't work on a Mac because it expects hera to be compiled as a `.so`)